### ISSUE TITLE
Not guessing virtual destructors

### ISSUE THREAD

**Issue by sei-eschwartz** [cmu-sei/pharos#105]
As discussesd in https://github.com/cmu-sei/pharos/issues/101#issuecomment-642337458 0x70d650 is almost definitely a constructor but is blocked because `possiblyVirtual(0x70d650)`.

Right now possibleVirtual is an opaque table that does not evolve, but it probably should.

Here is a snippet from the log (not completed), but I suspect we will never conclude that 0x70d650 is present in a vftable:
```
cat cncm-acorn.results-ed.log | fgrep 0xa2c85c
reasonVFTable1(0xa2c85c).
Concluding factVFTable(0xa2c85c).
reasonVFTable1(0xa2c85c).
Concluding factVFTableWrite(0x70d682, 0x70d650, 0, 0xa2c85c).
Concluding factVFTableWrite(0x70d133, 0x70d0a0, 0, 0xa2c85c).
reasonVFTable1(0xa2c85c).
Concluding factVFTableEntry(0xa2c85c, 0, 0x84deb0).
reasonVFTableSizeGTE_A(0xa2c85c, 0x4).
Concluding factVFTableSizeGTE(0xa2c85c, 0x4).
Processing trigger fact... factVFTableWrite(0x70d133, 0x70d0a0, 0, 0xa2c85c)
Processing trigger fact... factVFTableWrite(0x70d682, 0x70d650, 0, 0xa2c85c)
Processing trigger fact... factVFTableEntry(0xa2c85c, 0, 0x84deb0)
Concluding factMethodInVFTable(0xa2c85c, 0, 0x84deb0).
reasonClassCallsMethod_B(0xa2c85c, 0x70d650, 0x84deb0).
reasonClassCallsMethod_B(0xa2c85c, 0x70d0a0, 0x84deb0).
reasonVFTable1(0xa2c85c).
reasonVFTable1(0xa2c85c).
```

So I would contend that we should force this into a guess, and eventually relax `possibleVirtual` as a result.

**Comment by sei-eschwartz** [comment]
My analysis turned out to be completely wrong.  0x70d650 is a destructor, but we lacked some basic rules to guess virtual destructors.

**Comment by sei-eschwartz** [comment]
Fixed in 4c5cea6
